Sources/Swift-Big-Number-Core.swift

```diff
@@ -1049,14 +1055,31 @@ public struct BInt:
 
 fileprivate extension String
 {
-	// Splits the string into equally sized parts (exept for the last one).
-	func split(_ count: Int) -> [String] {
-		return stride(from: 0, to: self.count, by: count).map { i -> String in
-			let start = index(startIndex, offsetBy: i)
-			let end = index(start, offsetBy: count, limitedBy: endIndex) ?? endIndex
-			return String(self[start..<end])
-		}
-	}
+	// FIXME: The *split* causes a massive overhead due to *self* calls. Please remove.
```

Comment:
If `split` isn't used anymore then please remove this extension entirely.

Reply:
Ok, will do.